frontend/src/components/TeamReview.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
This component should be a part of the ADMIN panel 
Which is v3 of Orbis
Comment it out for now
We'll focus on v1 for now - 
v1 = Event creation by admin/organizers + All submodules required for Event Creation
v2 = User registration and Even Marketplace
v3 = Admin Panel